app/services/import-state.js

```diff
@@ -0,0 +1 @@
+export { default } from "ember-ebau-gwr/services/import-state";
```

Comment:
You can delete this file. With an engine, you only export files of which you are sure they are needed by a host app.

Reply:
done :+1: